### PR TITLE
Demo survey populate script

### PR DESCRIPTION
**Pull request by sturmer-eha**
Things look good within Parse Dashboard
Please feel free to extend this branch with more questions by putting them into DemoData.js

The web admin somehow does not show me more than one form per survey even if I see multiple forms within mongodb

### CODE REVIEW

File: tool-populate/data/DemoData.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+const DemoData = {
+
+  startDate: timestampToFormattedDate( Date.now() ),
+
+  endDate: timestampToFormattedDate( Date.now() + 604800*1000 ), // + one week from now
+
+  surveys: [
+    {
+      title: 'Demo Survey #1',
+      title: 'Demo survey for testing various things',
```
Comment (dan-nyanko) [inline tool-populate/data/DemoData.js]: should this be description?

**Comment by dan-nyanko** [comment]
@sturmer-eha 
I'm trying to populate our hosted parse server but get the error:

```
Creating Parse server data...
Failed to create demo Survey, error code: Permission denied for this action.
```

**Comment by sturmer-eha** [comment]
I have tested it on the local server only (`.scripts/run-parse-server.sh`), did not touch the common database since it's in constant use. The remote server should technically behave the same, but let me see what I can dig up.

**Comment by sturmer-eha** [comment]
Should be good now for filling in data on a remote server, didn't feel like wiping the whole thing to test the `-r` part. Should be the same `useMasterKey` thing.

File: tool-populate/api/Forms.js [hunk 2]
```diff
@@ -43,9 +49,41 @@ function createForms(parentSurvey) {
...
+  newForm.set('title', 'Form ' + Date.now())
+  newForm.set('order', 1)
+  newForm.set('deleted', false)
+
+  newForm.save(null, {
+    useMasterKey: true,
+    success: function(response) {
+      if (parentSurvey) {
+        var relation = parentSurvey.relation('forms')
+        relation.add(newForm)
```
Comment (dan-nyanko) [inline tool-populate/api/Forms.js:76]: Does newForm have an objectId?  If not this may need set to the `response`

**Pull request merged**